Change "video" to "medium", as the word should cover both audio and video (#503)
sveltejs/learn.svelte.dev: content/tutorial/02-advanced-svelte/05-bindings/03-media-elements/README.md

The complete set of bindings for `<audio>` and `<video>` is as follows — seven _readonly_ bindings...

> - `duration` (readonly) — the total duration of the media, in seconds

Reviewer: actually, i really don't like how this reads. gonna just rework it

```suggestion
- `duration` (readonly) — the total duration, in seconds
```

> - `duration` (readonly) — the total duration of the medium, in seconds

Reviewer: ```suggestion
- `duration` (readonly) — the total duration of the media, in seconds
```

...and five _two-way_ bindings:

> - `currentTime` — the current point in the media, in seconds
> - `playbackRate` — how fast to play the media, where `1` is 'normal'

Reviewer: ```suggestion
- `currentTime` — the current position of the playhead, in seconds
- `playbackRate` — speed up or slow down (`1` is 'normal')
```

> - `currentTime` — the current point in the medium, in seconds
> - `playbackRate` — how fast to play the medium, where `1` is 'normal'

Reviewer: ```suggestion
- `currentTime` — the current point in the media, in seconds
- `playbackRate` — how fast to play the media, where `1` is 'normal'
```
